### PR TITLE
Added IE hack style for focus.

### PR DESCRIPTION
**Pull request by tmccoy529**
Added new IE hack to fix focus issues.

### CODE REVIEW

**Review by martypowell — APPROVED** [review]

**Review by martypowell — CHANGES_REQUESTED** [review]
@tmccoy529 the ms-focus-within needs to go inside the media query in ie-hacks

File: src/styles/overrides/_ie-hacks.scss [hunk 1]
```diff
@@ -26,3 +26,6 @@
     background: $pale-blue;
   }
 }
+.ms-focus-within {
```
Comment (martypowell) [inline src/styles/overrides/_ie-hacks.scss]: This needs to move into the media query, otherwise it will be available in all browsers.

**Review by martypowell — DISMISSED** [review]
Looks good

**Comment by tmccoy529** [comment]
@martypowell the class was moved inside of the media query

**Comment by martypowell** [comment]
@tmccoy529 can you bump the version number in the package.json file for this, and then merge this pull request so i can push up to npm?

**Review by martypowell — APPROVED** [review]

**Comment by martypowell** [comment]
@tmccoy529 merged on behalf you so, so we can get this up to npm.

**Pull request merged**